assets/tiled_object_template.tmx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<map version="1.4" tiledversion="1.4.2" orientation="orthogonal" renderorder="right-down" width="3" height="3" tilewidth="32" tileheight="32" infinite="0" nextlayerid="3" nextobjectid="3">
```

Comment:
Wow, it's been a couple years heh

Reply:
Yeah, though even at the time this file was added, Tiled 1.8 was already out. So the file appeared older than it actually was. :-)